<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputDialogDemo</class>
 <widget class="QWidget" name="InputDialogDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input Dialog demo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout1">
     <item>
      <widget class="QPushButton" name="btnChooseFromList">
       <property name="text">
        <string>Choose from list</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditList"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout2">
     <item>
      <widget class="QPushButton" name="btnGetName">
       <property name="text">
        <string>get name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout3">
     <item>
      <widget class="QPushButton" name="btnEnterInteger">
       <property name="text">
        <string>Enter an integer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditInteger"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nama : Muhammad Daffa Dzaki Ahnaf</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Nim : F1D022142</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Kelas : C</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
